src/gpu/intel/jit/ir/tensor.hpp

```diff
@@ -458,9 +417,9 @@ class layout_t {
                 && self_blocks[i].stride == other_blocks[i].stride
                 && other_blocks[i].block % self_blocks[i].block == 0);
     }
-    bool operator>=(const layout_t &other) const { return other <= *this; }
 
-    bool is_equal(const layout_t &other, bool compare_offset = true) const {
+    bool is_isomorphic(
```

Comment:
Nit: isomorphic may be a bit broad, e.g. "isomorphism" may assume we don't care about whether dimensions (`pvar_t` objects) match between layouts

Another option: use something referring to "equivalent", e.g. `is_equiv()`
NumPy has [array_equiv](https://numpy.org/doc/2.1/reference/generated/numpy.array_equiv.html#numpy-array-equiv)

@rjoursler @atkassen @Simonsays095 What do you think?

Reply:
My main issue with `is_equiv` is it is unclear what notion of equivalency is used (for example, the broad interpretation is just as much of an isomorhpism is just as much an equivalency relationship).  How about `norm_eq` which directly references the `normalization` routine?

Reply:
Alternatively, you could just remove `is_equal` and use `layoutA.normalize().is_strictly_equal(layoutB.normalize())` in all cases where this is currently used, which only appears to be a couple of cases. It's a bit more verbose, but this conversation seems to indicate that verbosity is probably welcome.

I would prefer this to the optional `representative` function idea, since you would be able to use `is_equiv` expecting one idea of equivalent and having the code use another, due to the default argument.